### PR TITLE
fix: pinyin in textarea

### LINKED ISSUE #28940: Input.TextArea组件，当输入的字符长度等于maxLength，继续输入（中文输入法）时，会出现英文替换字符后几位
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/daizishutishidewenbenyu-antd4103-forked-invjq?file=/index.js:295-302)

### Steps to reproduce
1.maxLength={20}  defaultValue="11111111112222222222"，文本长度为20；
2.Input.TextArea获取焦点进行输入，切换到拼音输入法，反复点击英文，会发现“22”被替换成了英文

### What is expected?
value文本长度等于maxLength时，之后无论怎么输入，都不应该修改value值，应保持“11111111112222222222”，而不是类似“111111111122222222ad”等替换。

### What is actually happening?
value文本长度等于maxLength时，之后输入中文，“11111111112222222222”会被替换成“111111111122222222ad”之类。

| Environment | Info |
|---|---|
| antd | 4.10.3 |
| React | 16.14.0 |
| System | windows 10 |
| Browser | Google Chrome 版本 87.0.4280.141（正式版本）64 |

### PR DESCRIPTION
**Pull request by HomyeeKing**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close #28940
### 💡 Background and solution

_针对textarea的进行中文输入法的修复，相对于#28734， 基本没有副作用，无需修改任何历史test case（因为那个包含了Input.tsx的修改 :/ ）_

_具体测试可以通过这个版本的codesandbox
通过这个修改， [排版的editable text with limited length](https://ant.design/components/typography-cn/#components-typography-demo-interactive)中存在的类似问题也得到解决_

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/typography/__tests__/index.test.js [hunk 1]
```diff
@@ -436,17 +436,17 @@ describe('Typography', () => {
...
+      //     // Not trigger when inComposition
+      //     wrapper.find('textarea').simulate('compositionStart');
+      //     wrapper.find('textarea').simulate('keyDown', { keyCode: KeyCode.ENTER });
+      //     wrapper.find('textarea').simulate('compositionEnd');
+      //     wrapper.find('textarea').simulate('keyUp', { keyCode: KeyCode.ENTER });
+
+      //     // Now trigger
+      //     wrapper.find('textarea').simulate('keyDown', { keyCode: KeyCode.ENTER });
+      //     wrapper.find('textarea').simulate('keyUp', { keyCode: KeyCode.ENTER });
+      //   });
```
Comment (HomyeeKing) [inline components/typography/__tests__/index.test.js:450]: 真的不知道为什么会报find 0 node这种错，我打印出 确实是存在的 ，而且下面的test和这个是类似的 只有这个没过

Comment (HomyeeKing) [reply components/typography/__tests__/index.test.js:450]: 还有下面的compile \ lint \ codesandbox 相关的也没通过，不知道为啥🤷 本地的compile lint都跑过了

**Comment by afc163** [comment]
@HomyeeKing 进度如何？

**Comment by HomyeeKing** [comment]
@afc163 当时只有test不通过了 但是这应该是enzyme的bug，有成员在跟进 后续问了 也还没有解决

**Comment by HomyeeKing** [comment]
相关issue https://github.com/enzymejs/enzyme/issues/2493

**Pull request closed without merging**